plugins/topology/src/utils/topology-utils.ts

```diff
@@ -26,6 +26,8 @@ export const WORKLOAD_TYPES: string[] = [
   ModelsPlural.daemonsets,
 ];
 
+export const VM_TYPES: string[] = [ModelsPlural.virtualmachines];
```

Comment:
I know we have WORKLOAD_TYPES added in the utils , which should have been in the `types`

Could you move `VM_TYPES` to `types -> vms.ts`